src/SectorView.cpp

```diff
@@ -47,6 +47,128 @@ enum DetailSelection {
 static const float ZOOM_SPEED = 15;
 static const float WHEEL_SENSITIVITY = .03f; // Should be a variable in user settings.
 
+#define FFRAC(_x) ((_x)-floor(_x))
```

Comment:
If you're going to move the definition of FFRAC to the top of the file, please remove it from its previous location :D Makes reviewing the file much easier, as it doesn't look like it was invented from whole cloth.

Reply:
Thanks, accidentally missed, corrected, thanks, inadvertently missed, corrected!